.github/workflows/build-documentation.yml

```diff
@@ -0,0 +1,33 @@
+# Workflow to build and publish latest documentation from abr/feature/build-documentation
+
+name: build ABR documentation
+on: 
+  push:
+    branches:
+      - feature/abr-documentation
+      
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-20.04
+    container: python:3.8
+    
+
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2  # checkout code
+      
+      - name: Setup project
+        run: pip install -r requirements.txt  # install additional requirements (among which sphinx theme)
+      
+      - name: Build-Docs
+        run: mkdocs build
+      
+      - name: Deploy Pages
+        # env:
```

Comment:
This part can be removed since we do not need secrets here

- [ ] Remove secrets